nilearn/plotting/find_cuts.py

```diff
@@ -209,6 +210,13 @@ def find_cut_slices(img, direction='z', n_cuts=12, spacing='auto'):
 
     data = _smooth_array(data, affine, fwhm='fast')
 
+    if (isinstance(n_cuts, numbers.Number) and n_cuts > 0):
+        pass
+    else:
+        message = ("The number of cuts in the direction '%s' must be "
+                   "an integer between 0 and %d" % (direction, this_shape))
```

Comment:
As @AlexandreAbraham suggested adding something like "you provided n_cuts=whatever" gives a better clue to the user as to what he did wrong.